cmd/kube-apiserver/app/server.go

```diff
@@ -299,7 +299,15 @@ func Run(s *options.ServerRunOptions) error {
 	}
 
 	admissionControlPluginNames := strings.Split(s.GenericServerRunOptions.AdmissionControl, ",")
-	pluginInitializer := kubeadmission.NewPluginInitializer(client, sharedInformers, apiAuthorizer)
+	var cloudConfig []byte
+
+	if s.CloudProvider.CloudConfigFile != "" {
+		cloudConfig, err = ioutil.ReadFile(s.CloudProvider.CloudConfigFile)
```

Comment:
How would that work for federation api server? which takes same set of arguments and initialzies admission controller in similar way.

Currently federation api server seems to be simply throwing away cloud config. options etc but it does accept them same as kube-apiserver.